src/android/com/megster/cordova/BluetoothSerial.kt

```diff
@@ -96,9 +92,29 @@ class BluetoothSerial : CordovaPlugin() {
         BluetoothSerialService.stop()
     }
 
-    private fun enableBluetoothIfNecessary() {
-        if (!bluetoothAdapter.isEnabled) {
-            bluetoothAdapter.enable()
+    private fun isEnabled(callbackContext: CallbackContext) {
+        try {
+            callbackContext.success(if (bluetoothAdapter.isEnabled()) 1 else 0)
```

Comment:
please dont right if statements like this

please write them like this:

if {
} else {
}

Reply:
are you saying people don’t use ternary when writing kotlin?

Reply:
are you saying people don’t use ternary when writing kotlin?

Reply:
im saying its unnecessarily difficult to read

Reply:
a 2 year old like me isn't that clever

Reply:
its fine - i just find it harder to read - wont block on it